### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by msluther**
This gets CI/travis back into a workable state after the webhook support was combined with the dynamodb work.

Still needs fixed:
- [x] One webhook test is still failing and needs to be addressed.

## Summary

Motivation: working CI

## Changelog

- Fix Travis CI

## Test Plan

Existing tests all pass on Travis

### CODE REVIEW

**Comment by msluther** [comment]
@JacopoDaeli This test is still failing.  Can you look at it:

```
1) Status-Handler
       integration
         should send build_started webhook and be robust:
     ValidationException: The number of conditions on the keys is invalid
```

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "pretest": "npm run swagger",
     "test": "nyc --reporter=lcov --reporter=text mocha --require test/setup test/*.test.js --timeout 10000",
     "lint": "eslint-godaddy *.js \"{preboots,routes,test}/**/*.js\"",
-    "localstack": "docker run -it -p 4569:4569 --rm localstack/localstack",
+    "localstack": "docker run -it -p 4569:4569 -e SERVICES=dynamodb --rm localstack/localstack -sharedDb",
```
Comment (msluther) [inline package.json]: Ooops. that wasn't needed, the `-sharedDb`

File: test/status-handler.test.js [hunk 2]
```diff
@@ -267,7 +272,7 @@ describe('Status-Handler', function () {
       // somewhere in the first second of its launch. Rather than force the
       // test order of execution, I have decided to be defensive against this
       // behavior by waiting a second and then cleaning up the tables.
-      await wait(1000);
+      await wait(5000);
```
Comment (msluther) [inline test/status-handler.test.js]: that wasn't necessary either.

File: test/status-handler.test.js [hunk 3]
```diff
@@ -23,6 +23,8 @@ describe('Status-Handler', function () {
     let status;
 
     before(() => {
+      const dynamoDriver = new AWS.DynamoDB(config.database);
+      dynamodb.dynamoDriver(dynamoDriver);
```
Comment (msluther) [inline test/status-handler.test.js]: derp. also not required.  🤦‍♂

**Comment by burgessa23** [comment]
LGTM

**Comment by JacopoDaeli** [comment]
@msluther I'd love to look into this, but I cannot run my tests locally anymore with this PR.

**Comment by JacopoDaeli** [comment]
`master` is still fine. only when I am checked out on this new code, tests do not run for me.

**Comment by JacopoDaeli** [comment]
image

This gets stocked.

**Comment by JacopoDaeli** [comment]
Overall, I am not sure to understand what's the root issue that makes the test currently failing in master.

**Comment by JacopoDaeli** [comment]
I see what's the blocking problem is. You must export `AWS_SECRET_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`. Pushing this now.

**Comment by JacopoDaeli** [comment]
@msluther tests should now passed both locally and on Travis. Please merge this today so I can rebase in my current PR.

**Pull request merged**